src/components/Settings.tsx

```diff
@@ -138,14 +167,15 @@ export default function Settings() {
      <p className="text-sm mb-6 text-green-600 font-bold">{showUserMessage && usermessage}</p>
       <label className="label sr-only" htmlFor="username">Name</label>
```

Comment:
For some reasons, the labels are not displayed. Please investigate.

Reply:
You mean "Name" and so on? Labels are not meant to be displayed (screen reader only).

Reply:
That is not correct, I think.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

The label for the language settings display just fine like I think they should. In any case, I would like to read on the page what each field is without relying on placeholders.